app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -98,10 +98,20 @@ def total_user_count_report
       @total_user_count_report ||= Reporting::TotalUserCountReport.new(report_date)
     end
 
+    def agency_and_sp_report
+      @agency_and_sp_report ||= Reporting::AgencyAndSpReport.new(report_date)
+    end
+
+    def monthly_active_users_count_report
+      @monthly_active_users_count_report ||= Reporting::MonthlyActiveUsersCountReport.new(
+        report_date,
+        )
+    end
+
     def active_users_count_report
       @active_users_count_report ||= Reporting::ActiveUsersCountReport.new(
         report_date,
-      )
+        )
```

Comment:
```suggestion
      )
```

Merge conflict mess that slipped by me.